examples/aws-exfiltration-protection/versions.tf

```diff
@@ -0,0 +1,14 @@
+# versions.tf
+terraform {
+  required_providers {
+    databricks = {
+      source  = "databricks/databricks"
+      version = ">=0.5.1"
```

Comment:
it makes sense to use something 1.x as minimal dependency. Let have it as a follow up task